finished-work/main.css

```diff
@@ -0,0 +1,126 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans')
+/* ---- reset ---- */
+
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  height: 0;
+}
+
+
+/* ---- typography ---- */
+
+body {
+  color: #111;
+  font-family: "Open Sans";
+  font-size: 16px;
+  line-height: 147%;
+}
+
+h1 {
+  
+}
+
+h2 {}
+
+h3 {}
+
+p {
+  margin: 0;
+}
+
+
+
+/* ---- layout ---- */
+
+#site-header {
+  padding: 30px 0;
+  color: white;
+  background: #111;
+}
+
+#site-footer {
+  color: white;
+  padding: 60px 0;
+  background: #111;
+}
+
+#site-main {
+  margin-bottom: 60px;
+}
+
+.container {
+  margin: 0 20px;
+}
+
+.section {
+  margin-bottom: 20px;
+  padding-top: 20px;
+}
+
+@media (min-width: 768px) {
+  .container {
+    max-width: 728px;
+    margin: 0 auto;
+  }
+}
+
+
+
+
+/* ---- components ---- */
+
+
+/* ---- had to look for the min-height thing to make the image appear as whole ---- */
+#recipe1 .recipe-picture { 
+  width:980px; 
+  min-height:500px; 
```

Comment:
A nice trick here to make the image display as a whole without using ```height``` would be to set the ```line-height``` to the desired image height as we saw